### PR TITLE
feat(cli): Create standalone imagery config and remove disabled layer. BM-810

### PR DESCRIPTION
**Pull request by Wentao-Kuang**
- This update the basemaps import pull request creation cli to create standalone imagery configs. 
- Another update is the disabled flag is not useful anymore once we using standalone configs, so I have removed them.

### CODE REVIEW

File: packages/cli/src/cli/cogify/action.make.cog.pr.ts [hunk 1]
```diff
@@ -20,8 +20,8 @@ export const DefaultCategorySetting: Record<Category, CategorySetting> = {
   [Category.Urban]: { minZoom: 14 },
   [Category.Rural]: { minZoom: 13 },
   [Category.Satellite]: { minZoom: 5 },
-  [Category.Event]: { disabled: true },
-  [Category.Other]: { disabled: true },
+  [Category.Event]: { minZoom: 32 },
```
Comment (Wentao-Kuang) [inline packages/cli/src/cli/cogify/action.make.cog.pr.ts:23]: Do we still need set `minZoom:32` for standalone imagery config?

File: packages/cli/src/cli/github/make.cog.pr.ts [hunk 2]
```diff
@@ -26,23 +35,39 @@ export class MakeCogGithub extends Github {
...
-    await fsa.write(tileSetPath, await this.formatConfigFile(tileSetPath, newTileSet));
+    if (disabled) {
+      // Prepare new standalone tileset config
+      const tileSet: TileSetConfigSchema = {
+        type: TileSetType.Raster,
+        id: ConfigId.prefix(ConfigPrefix.TileSet, layer.name),
+        title: layer.title,
+        layers: [layer],
+      };
+      const tileSetPath = fsa.joinAll(this.repoName, 'config', 'imagery', `${layer.name}.json`);
```
Comment (Wentao-Kuang) [inline packages/cli/src/cli/github/make.cog.pr.ts:60]: Could add to directory to `config/tileset/individual/name.json`.

File: packages/cli/src/cli/cogify/action.make.cog.pr.ts [hunk 3]
```diff
@@ -67,8 +66,8 @@ export class CommandCogPullRequest extends CommandLineAction {
       defaultValue: 'linz/basemaps-config',
       required: false,
     });
-    this.disabled = this.defineFlagParameter({
-      parameterLongName: '--disabled',
+    this.addToAerial = this.defineFlagParameter({
+      parameterLongName: '--add_to_aerial',
```
Comment (blacha) [inline packages/cli/src/cli/cogify/action.make.cog.pr.ts]: why `_` ? 

What is the default here?

File: packages/cli/src/cli/github/make.cog.pr.ts [hunk 4]
```diff
@@ -49,7 +49,7 @@ export class MakeCogGithub extends Github {
     this.getBranch(branch);
 
     this.logger.info({ imagery: this.imagery }, 'GitHub: Get the master TileSet config file');
-    if (disabled) {
+    if (addToAerial || category === Category.Event || category === Category.Other) {
```
Comment (blacha) [inline packages/cli/src/cli/github/make.cog.pr.ts:52]: this is inverted? `if (addToAreal) { proceeds to not add to aerial }` ?

File: packages/cli/src/cli/cogify/action.make.cog.pr.ts [hunk 5]
```diff
@@ -13,15 +13,14 @@ export enum Category {
...
-  disabled?: boolean;
 }
 
 export const DefaultCategorySetting: Record<Category, CategorySetting> = {
   [Category.Urban]: { minZoom: 14 },
   [Category.Rural]: { minZoom: 13 },
   [Category.Satellite]: { minZoom: 5 },
-  [Category.Event]: { minZoom: 32 },
-  [Category.Other]: { minZoom: 32 },
+  [Category.Event]: {},
```
Comment (blacha) [inline packages/cli/src/cli/cogify/action.make.cog.pr.ts:22]: we should add a `addToAerial: false` here ?

File: packages/cli/src/cli/github/make.cog.pr.ts (hunk 4 above)
Comment (Wentao-Kuang) [reply packages/cli/src/cli/github/make.cog.pr.ts]: Good spot. Just fixed.

File: packages/cli/src/cli/cogify/action.make.cog.pr.ts (hunk 5 above)
Comment (Wentao-Kuang) [reply packages/cli/src/cli/cogify/action.make.cog.pr.ts]: Added.

File: packages/cli/src/cli/github/make.cog.pr.ts [hunk 6]
```diff
@@ -26,23 +35,39 @@ export class MakeCogGithub extends Github {
...
-    // Prepare new aerial tileset config
     this.logger.info({ imagery: this.imagery }, 'GitHub: Get the master TileSet config file');
-    const tileSetPath = fsa.joinAll(this.repoName, 'config', 'tileset', `${filename}.json`);
-    const tileSet = await fsa.readJson<ConfigTileSetRaster>(tileSetPath);
-    const newTileSet = await this.prepareRasterTileSetConfig(layer, tileSet, category);
-
-    // skip pull request if not an urban or rural imagery
-    if (newTileSet == null) return;
-    await fsa.write(tileSetPath, await this.formatConfigFile(tileSetPath, newTileSet));
+    if (addToAerial) {
```
Comment (blacha) [inline packages/cli/src/cli/github/make.cog.pr.ts:52]: This is not always going to be `aerial` maybe we should call it `--individual` ?

**Review by blacha — APPROVED** [review]

**Pull request merged**